task_03.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""This docstring is a for loop to count strings"""
+
+
+def lexicographics(to_analyze):
+
+
+    """ A function that calculates the maximun, minimum and average.
+
+    Args:
+        to_analyze(mixed): list of strings.
+
+    Examples:
+       >>> lexicographics(data.SHAKESPEARE)
+       (12, 5, float('8.14'))
+    """
+    mylist = []
+    lines = to_analyze.split("\n")
+    for line in lines:
+        num_words = line.split()
+        mylist.append(len(num_words))
+    mymin = min(mylist)
+    mymax = max(mylist)
+    myav = float(sum(mylist))/float(len(mylist))
+    return mymax, mymin, myav
```

Comment:
```
- (12, 5, 8.14)
+ (12, 5, Decimal('8.14'))
```

The error message here is pretty explicit -- it expected you to return a decimal and you returned a float instead. It was otherwise a great implementation.